docs/dev/compiler-toolchain/solidity.md

```diff
@@ -142,18 +142,17 @@ This output format is utilized in Yul and LLVM IR compilation modes. These modes
   Cannot be used with the combined and standard JSON modes.  
 
 - `--llvm-ir`  
-  Switch to the LLVM IR mode.  
+  Switch to LLVM IR mode.  
   Only one input LLVM IR file is allowed.  
-  Cannot be used with the combined and standard JSON modes.  
+  Cannot be used with combined and standard JSON modes.  
 
 - `--force-evmla`  
-  Force switch to the EVM legacy assembly pipeline.  
-  Useful for older revisions of `solc` 0.8, where Yul was considered highly experimental and contained more bugs than today.  
+  Force use of the EVM legacy assembly pipeline.  
+  Useful for early versions of `solc` 0.8.x, where Yul was considered highly experimental and contained more bugs than today.  
 
 - `--system-mode`  
-  Enable the system contract compilation mode.  
-  In this mode, zkEVM extensions are enabled. For example, calls to addresses `0xFFFF` and below are substituted by special  
-  zkEVM instructions. In the Yul mode, the `verbatim_*` instruction family is available.  
+  Enable system contract compilation mode.  
+  In this mode, zkEVM extensions for system contracts are enabled. For example, calls to addresses `0xFFFF` and below are substituted by special zkEVM instructions. In Yul mode, the `verbatim_*` instruction family is available.  
```

Comment:
Let's drop 'for system contracts'.
There are several instructions available in Yul mode unrelated to system contracts, like 'throw'.
I see that the flag name is outdated, but we should keep it for compatibility until we have moved to our fork of 'solc'.